src/render/OpenGL.cpp

```diff
@@ -320,6 +320,10 @@ void CHyprOpenGLImpl::renderRectWithDamage(wlr_box* box, const CColor& col, pixm
     RASSERT((box->width > 0 && box->height > 0), "Tried to render rect with width/height < 0!");
     RASSERT(m_RenderData.pMonitor, "Tried to render rect without begin()!");
 
+    // TODO: can we early out here?
```

Comment:
yeah, but the damage won't be empty since we're rendering a frame.

Reply:
will it never be empty?
And if so, why are you checking it later in the function?

Reply:
oh, dunno, maybe a leftover